pinot-core/src/main/java/org/apache/pinot/core/operator/transform/TransformOperator.java

```diff
@@ -95,6 +96,14 @@ public TransformResultMetadata getResultMetadata(ExpressionContext expression) {
     return _transformFunctionMap.get(expression).getResultMetadata();
   }
 
+  public NullValueVectorReader getNullValueVectorReader(ExpressionContext expression) {
```

Comment:
This method should be in the projection layer instead of the transform layer. In the transform layer, we should be able to resolve the null values for the expression, instead of the column